solidity/contracts/BTCUtils.sol

```diff
@@ -27,13 +27,17 @@ library BTCUtils {
     /// @param _flag    The first byte of a VarInt
     /// @return         The number of non-flag bytes in the VarInt
     function determineVarIntDataLength(bytes memory _flag) internal pure returns (uint8) {
-        if (uint8(_flag[0]) == 0xff) {
+        return determineVarIntDataLengthAt(_flag, 0);
+    }
+
+    function determineVarIntDataLengthAt(bytes memory _b, uint256 _at) internal pure returns (uint8) {
```

Comment:
Could we add a brief description to this function (may be similar to the `determineVarIntDataLength`)?